internal/exec/util/unit.go

```diff
@@ -110,8 +110,8 @@ func (ut Util) MaskUnit(unit types.Unit) (string, error) {
 	return filepath.Join("/", SystemdUnitsPath(), unit.Name), nil
 }
 
-func (ut Util) EnableUnit(unit types.Unit) error {
-	return ut.appendLineToPreset(fmt.Sprintf("enable %s", unit.Name))
+func (ut Util) EnableUnit(enabledUnits string) error {
+	return ut.appendLineToPreset(fmt.Sprintf("enable %s", enabledUnits))
```

Comment:
`enabledUnits` is plural while `disabledUnit` (from below) is not. Let's choose either plural or not for both of them.

Reply:
Fixed